components/ruby/lib/license_acceptance/strategy/file.rb

```diff
@@ -77,12 +77,12 @@ def persist(product_relationship, missing_licenses)
 
       def persist_license(folder_path, product, parent, parent_version)
         path = ::File.join(folder_path, product.filename)
-        logger.info("Persisting a license for #{product.name} at path #{path}")
+        logger.info("Persisting a license for #{product.pretty_name} at path #{path}")
         ::File.open(path, ::File::WRONLY | ::File::CREAT | ::File::EXCL) do |license_file|
           contents = {
-            name: product.name,
+            name: product.pretty_name,
```

Comment:
This will change what is persisted in the marker files. Users who have already accepted the InSpec license will have `name: inspec` instead of `name: InSpec` but since we don't read the contents of that file anywhere yet I figured the best time to make this change was now. Anyone disagree?

I could also persist `product.id` instead of `product.pretty_name` so it is easier to change the pretty name in the future. It would keep that field purely UX focused. Which as I type that out I realize is the right thing to do, so I'm going to make that change.

Reply:
Changed this to persist both the product id and product name